evap/evaluation/tests/test_models.py

```diff
@@ -537,6 +538,21 @@ def test_textanswer_review_state(self):
             evaluation.TextAnswerReviewState.REVIEWED,
         )
 
+    def test_state_change_log_translated(self):
+        evaluation = baker.make(
+            Evaluation,
+            state=Evaluation.State.NEW,
+        )
+        evaluation.save()
+        with translation.override("de"):
+            # Refresh reference to evaluation, to force new log entry
+            evaluation = Evaluation.objects.get(pk=evaluation.pk)
+            evaluation.ready_for_editors()
+            evaluation.save()
+            log_entry = evaluation.related_logentries().order_by("id").last()
+            self.assertEqual(Evaluation.state_to_str(log_entry.data["state"]["change"][0]), "neu")
+            self.assertEqual(Evaluation.state_to_str(log_entry.data["state"]["change"][1]), "vorbereitet")
```

Comment:
I was kind of hoping that we could have a bit of a larger frame in the test -- this test now is very low-level and wouldn't catch if we someday stopped calling `state_to_str` in `transform_log_action`. A view level test might be more appropriate here. Sorry, when I said a model test would be fine I was hoping that there is a higher-level method on the model that could be used, something like "give me the list of log entries as strings to be shown on the page".

I can take over the testing from here if you want, ping me in case you'd prefer that.

Reply:
Would this test fit in `TestEvaluationEditView`? My idea would be to use similar logic as in in the model test and use some kind of `self.assertIn("neu", page)` or `self.assertIn("neu", log_div)` and also the negation if "en" is selected.
If that would be the path you indended, I would try doing it myself, and reach out again when the next roadblock appears

Reply:
Yes, that sounds good 👍 . "neu" itself might appear in other places on the page, so `self.assertIn("neu", page)` might pass even if your translation was broken. Searching in the log-div would probably be fine if the word doesn't appear there otherwise (should be asserted in the test then), or you could add a bit more context around, i.e. search for "Zustand geändert: X --> neu" or something (don't currently know the exact formatting / wording used in the log entry).

Reply:
See [44cb0c7](https://github.com/e-valuation/EvaP/pull/2071/commits/44cb0c7a314ebcf9867b47a28aae07f05d00fbda)